ephios/settings.py

```diff
@@ -296,3 +300,13 @@ def GET_SITE_URL():
     if site_url.endswith("/"):
         site_url = site_url[:-1]
     return site_url
+
+
+REST_FRAMEWORK = {
+    "DEFAULT_PERMISSION_CLASSES": ["rest_framework.permissions.DjangoModelPermissions"],
```

Comment:
didn't test it, but according to the note in https://www.django-rest-framework.org/api-guide/permissions/#djangoobjectpermissions we should be using https://github.com/rpkilby/django-rest-framework-guardian

Reply:
So I got the view permissions working. Here's the tricky details:

* DjangoModelPermissions checks for Model permissions. That can't be what we want, as we are also using object permissions.

* DjangoObjectPermissions checks if a user has model **and** object permissions, but not for **viewing**. Doesn't sound like what we want, but it actually is. See the next point.

* The CustomDjangoObjectPermissions shown in the django-rest-framework-guardian readme adds view permissions to the default ObjectPermission by DRF. Point is though, we don't normally give users view permissions for the event model. They wouldn't see the events they have object permissions for. To still achieve correct filtering, we use the DRF-guardian filter on the view.